content/docs/setup/kubernetes/upgrade/notice/index.md

```diff
@@ -1,36 +1,20 @@
-- Addons are no longer exposed via separate load balancers.  Instead addons can now be optionally exposed via the Ingress Gateway.  To expose an addon via the Ingress Gateway, please follow the [Remotely Accessing Telemetry Addons](/docs/tasks/telemetry/gateways/) guide.
-- The built-in Istio Statsd collector has been removed. Istio retains the capability of integrating with your own Statsd collector, using the `--set global.envoyStatsd.enabled=true` flag.
-- The `ingress` series of options for configuring a Kubernetes Ingress have been removed.  Kubernetes Ingress is still functional and can be enabled using the `--set global.k8sIngress.enabled=true` flag.  Check out the [Securing Kubernetes Ingress with Cert-Manager](/docs/tasks/traffic-management/ingress/ingress-certmgr/) for how to secure your Kubernetes ingress resources.
-
-## Traffic Management
-
-- Outbound traffic policy now defaults to `ALLOW_ANY`.  Traffic to unknown ports will be forwarded as-is. Traffic to known ports (e.g., port 80) will be matched with one of the services in the system and forwarded accordingly.
-- During sidecar routing to a service, destination rules for the target service in the same namespace as the sidecar will take precedence, followed by destination rules in the service’s namespace, and finally followed by destination rules in other namespaces if applicable.
-- We recommend storing gateway resources in the same namespace as the gateway workload (e.g., `istio-system` in case of `istio-ingressgateway`).  When referring to gateway resources in virtual services, use the namespace/name format instead of using `name.namespace.svc.cluster.local`.
-- The optional egress gateway is now disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles by default.  If you need to control and secure your outbound traffic through the egress gateway, you will need to enable `gateways.istio-egressgateway.enabled=true` manually in any of the non-demo profiles.
-
-## Policy & Telemetry
-
-- `istio-policy` check is now disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles.  This change is only for `istio-policy` and not for `istio-telemetry`.  In order to re-enable policy checking, run `helm template` with `--set global.disablePolicyChecks=false` and re-apply the configuration.
-- The Service Graph component has now been deprecated in favor of [Kiali](https://www.kiali.io/).
-
-## Security
+- Most mixer CRDs were removed from the system to consolidate the CRD definitions in one data model.
+  In the event you are using a mixer plugin that has not transitioned to the new mixer data model,
```

Comment:
@douglas-reid can u pls confirm this only applies if user is using the old mixer model for their plugin?

Reply:
As I confirmed elsewhere, these flags only matter if you are still using the deprecated configuration schemas for Mixer.

Reply:
@sdake I chatted with Doug offline, so this helm flag is only needed when users aren't using any of the 6 CRDs:  


-  adapter, attributemanifest, and template for Istio extension declarations;
- handler, rule, and instance for specifying extension invocations.

Reply:
@linsun - do you want me to add that level of detail to the notice?  

Something like
"In the event you are using any of the Mixer CRDs that are not `adapter`, `attributemanifest`, `template` for Istio declarations; or `handler`, `rule, and `instance` for specifying extension invocations, this flag must be used.

Reply:
Yes, that would be awesome!